src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -1698,9 +1702,9 @@ public CompactionManager.AllSSTableOpStatus verify(Verifier.Options options) thr
     /**
      * Rewrites all SSTables according to specified parameters
      *
-     * @param skipIfCurrentVersion - if {@link true}, will rewrite only SSTables that have version older than the current one ({@link BigFormat#latestVersion})
+     * @param skipIfCurrentVersion - if {@link true}, will rewrite only SSTables that have version older than the current one ({@link org.apache.cassandra.io.sstable.format.big.BigFormat#latestVersion})
```

Comment:
opportunistic bugfix, without this IDEA marks it errorneous hence all class

Reply:
👍